spec/models/user_spec.rb

```diff
@@ -1,31 +1,16 @@
-require "rails_helper"
+require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  subject {
-    described_class.new(username: "username1",
-                        email: "username1@email.com",
-                        name: "Firstusername",
-                        lastname: "Firstuserlastname",
-                        password: "username1pass",
-                        password_confirmation: "username1pass")
-  }
+  
+  subject { create(:user) }
 
-  describe "Validations" do
-    it "is valid with valid attributes" do
-      expect(subject).to be_valid
-    end
-
-    it "is not valid without a password" do
-      subject.password = nil
-      expect(subject).to_not be_valid
-    end
-
-    it "is not valid without an email" do
+  describe 'Validations' do
+    it 'is not valid without an email' do
       subject.email = nil
       expect(subject).to_not be_valid
     end
 
-    it "is not valid without an username" do
+    it 'is not valid without an username' do
```

Comment:
Usa shoulda matcher:
```suggestion
    it { is_expected.to validate_presence_of(:username) }
    it { is_expected.to validate_uniqueness_of(:username) }
```

Reply:
Si, en efecto shoulda matchers se usa en el PR 36.